test/commands/iteration/test_run/process_test.rb

```diff
@@ -79,6 +79,7 @@ class Iteration::TestRun::ProcessTest < ActiveSupport::TestCase
 
     IterationChannel.expects(:broadcast!).with(iteration)
     IterationsChannel.expects(:broadcast!).with(iteration.solution)
+    TestRunChannel.expects(:broadcast!).with { |test_run| test_run == iteration.test_runs.last }
```

Comment:
Is this the way in which I can verify an expected value _after_ it has run? I couldn't just use `.with(iteration.test_runs.last)` like with the other calls, as the actual test run is created in the `Iteration::TestRun::Process.` call.

Reply:
This is absolutely fine. You could also do 
```
TestRunChannel.expects(:broadcast!).with(kind_of(Iteration::TestRun))
```

As you're not validating that the test run is particularly the "correct" test run, the `kind_of` check is probably just as safe. I don't have a preference over either. You could firm up the test by asserting that  `iteration.test_runs.size == 1` too